check_if_nginx_install.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/bash
+$1=rpm -q
+$2=bool
+if [[ $1 && $2  ]]
+then
+    echo "$1 installed"
+    sudo  yum -y autoremove
+else
+        sudo yum -y install nginx
+        sudo service nginx start
+        sudo systemctl enable nginx
```

Comment:
Are you using service or systemctl? Choose one of them. 
You should consider enabling the service and than starting it.